test/regression/ode_adaptive_tests.jl

```diff
@@ -126,6 +126,14 @@ sol_linear = solve(prob_linear, ESDIRK547L2SA2())
 sol_lorenz = solve(prob_lorenz, ESDIRK547L2SA2())
 @test length(sol_lorenz.u) < 1000
 
+# ESDIRK659L2SA
+
+sol_linear = solve(prob_linear, ESDIRK659L2SA())
+@test length(sol_linear.u) < 10
```

Comment:
I tested this method before, and it converges as expected, but the regression tests always failed(Yeah, I checked the adaptivity parameters and I can sure they are correct), really confusing🤔

see

![image](https://user-images.githubusercontent.com/52615090/231329598-ade8a6fc-5745-4ca6-968e-34f42c5a4b0c.png)